commons/src/main/java/fr/free/nrw/commons/MediaWikiImageView.java

```diff
@@ -151,7 +151,7 @@ private void loadImageIfNecessary(final boolean isInLayoutPass, final boolean tr
         // from the network.
         ImageContainer newContainer = mImageLoader.get(mUrl,
                 new ImageListener() {
-                    @Override
```

Comment:
Hm, I removed it because the IDE gives me the error: "@Override is not allowed when implementing interface method"